### ISSUE TITLE
Problems with Facebook inviter

### ISSUE THREAD

**Issue by loarie** [inaturalist/inaturalist#627]
not a priority IMO, but just wanted to log these bugs:
http://www.inaturalist.org/photos/inviter -> Facebook
used to allow you to search all your friends photos, I think something changed with the permissions Facebook is granting iNat to access Friends and Friends photos.
After disconnecting, deleting the iNat app from FB, and reconnecting my FB account from iNat, I can't see any friends (before I could see a subset of friends, maybe those with the iNat app installed??)

I also can't see any of my group's photos, which used to work

also last we checked you couldn't see page's photos, but that would be useful as seems like most of these natural history communities on facebook use pages not groups

**Comment by carrieseltzer** [comment]
Thanks Scott. I just linked my Facebook account to look at this and it says I don't have any friends yet when I try to see friends photos. See screen shot.

![screen shot 2015-07-08 at 1 44 46 pm](https://cloud.githubusercontent.com/assets/7501183/8577807/26a16e44-2579-11e5-9225-447f115ec790.png)


Also, I think groups are still more useful to connect to than pages, since pages tend to be one or a few people posting (like the AfriBats page--https://www.facebook.com/pages/AfriBats/639656049404160) while groups are lots of people posting (like the Slime Mold Identification and Appreciation group-- https://www.facebook.com/groups/SlimeMold/?ref=browser).

**Comment by loarie** [comment]
After a complaint from someone who can no longer add their photos from FB, I realized that FB now requires apps that access more than public profile and email to be reviewed to access things like photos. I submitted the review - takes 5 buisiness days. But I believe we'll have to set up a separate app for thing like viewing friends photos and group photos and posting (all things the inviter does) so will cross this bridge after seeing how the above goes. But these permissions alone might fix the 'bugs'
